etc/fixtures/add_user.js

```diff
@@ -0,0 +1,32 @@
+// Declare the connection URI
+const uri = 'mongodb://localhost:27017/rosalution_db';
```

Comment:
Having gone back to look at the ticket, it looks like these parameters were requested to be configurable arguments, if the user has their database at a different location, like we will on Azure, it would be helpful to be able to define where that database is and the script will interpret them. 

Having this feature would necessitate a usage comment at the top of the file to determine which order things should go in.

Reply:
Good catch. I believe I have made this change.

Reply:
This has been changed and now accepts arguments.